chsdi/templates/htmlpopup/bazl_intrinsisches_bodenrisiko.mako

```diff
@@ -0,0 +1,12 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+<tr>
+  <td class="cell-left">
+    ${_('ch.bazl.intrinsisches-bodenrisiko_sora.density_pop_km2')}
+  </td>
+  <td>
+    ${c['attributes']['density_pop_km2']}
```

Comment:
is there a constraint in the db - not NULL?

Reply:
good point I will add a check here in a new PR. In the mean time the legend has also changed.